sensing_launch/launch/aip_s1/pointcloud_preprocessor.launch.py

```diff
@@ -0,0 +1,108 @@
+
+# Copyright 2020 Tier IV, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import launch
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+
+
+def generate_launch_description():
+
+  pkg = 'pointcloud_preprocessor'
+
+  # set concat filter as a component
+  concat_component = ComposableNode(
+      package=pkg,
+      plugin='pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent',
+      name='concatenate_data',
+      remappings=[('/output', 'concatenated/pointcloud')],
+      parameters=[
+          {
+              'input_topics': ['/sensing/lidar/top/outlier_filtered/pointcloud',
+                               '/sensing/lidar/left/outlier_filtered/pointcloud',
+                               '/sensing/lidar/right/outlier_filtered/pointcloud'],
+              'output_frame': 'base_link',
+          }
+      ]
+  )
+
+  # set crop box filter as a component
+  cropbox_component = ComposableNode(
+      package=pkg,
+      plugin='pointcloud_preprocessor::CropBoxFilterComponent',
+      name='crop_box_filter',
+      remappings=[
+          ('/input', 'concatenated/pointcloud'),
+          ('/output', "mesurement_range_cropped/pointcloud"),
+          ('/min_z', '/vehicle_info/min_height_offset'),
+          ('/max_z', '/vehicle_info/max_height_offset'),
+      ],
+      parameters=[
+          {
+              'input_frame': 'base_link',
+              'output_frame': 'base_link',
```

Comment:
These must be 
```
'input_frame', LaunchConfiguration('base_frame'),
'output_frame': LaunchConfiguration('base_frame'),
```

as in https://github.com/tier4/autoware_launcher.iv.universe/blob/master/sensing_launch/launch/velodyne_VLS128.launch#L69

Reply:
done in cc23b23a0226dc702b2658dbdfd1e5c4decfa0f5. Thanks for mentioning, I missed that amidst the many changes

Reply:
@fred-apex-ai I noticed that we have to declare the arguments first in order to use LaunchConfiguration.
So we have to have `DeclareLaunchArgument('base_frame', default_value='base_link')` at the beginning of the launch file.
You would also have to import DeclareLaunchArgument from launch.actions modules.

Reply:
@mitsudome-r I have made that change and also updated `lidar.launch.xml` to pass that launch argument but I left it as a default. You want to be able to set it somewhere to pass it, where would you do that? Just adding a launch argument and leaving it as default at the same value as before doesn't get you anything, right? Please  check 3a8b54a0a7e6dca3d1fce5af0bc86c09e7c4323e

Reply:
The same question holds for `velodyne*.launch.xml`. They could modify `base_frame` in the past but it looked as though nobody did so I just coded it up in `velodyne_node_container.launch.py` as default